weave-js/src/components/PagePanelComponents/Home/Home.tsx

```diff
@@ -265,24 +265,26 @@ const HomeComp: FC<HomeProps> = props => {
         {/* Center Content */}
         {!loading && (
           <CenterSpace>
-            {props.browserType === 'recent' ? (
-              // This should never come up
-              <Placeholder />
-            ) : props.browserType === 'wandb' ? (
-              <CenterEntityBrowser
-                navigateToExpression={navigateToExpression}
-                setPreviewNode={setPreviewNode}
-                entityName={params.entity ?? ''}
-              />
-            ) : props.browserType === 'local' ? (
-              <CenterLocalBrowser
-                navigateToExpression={navigateToExpression}
-                setPreviewNode={setPreviewNode}
-              />
-            ) : (
-              // This should never come up
-              <Placeholder />
-            )}
+            <ErrorBoundary key={pathname}>
```

Comment:
Recommend "hide whitespace" while reviewing - the inner part here hasn't changed, just indented.